### PR TITLE
NEPT-1252: Release 2.4: QA feedback application for blockers

### PR DESCRIPTION
**Pull request by voidtek**
### Description

Release 2.4: QA feedback application for blockers

### Change log

Changed: NEPT-1271: Add behat test for last_update module and update the readme.
Changed: NEPT-1267: Move helper functions outside of install file.
Changed: NEPT-1268: Locked metatag files.
Changed: NEPT-1269: Move helper functions outside of install file.
Changed: NEPT-1270: Clean tpl for the nexteuropa_lastupdate module.
Changed: NEPT-1270: Refactor the preprocess method.

### Commands

```sh
drush cc all

```

### CODE REVIEW

File: profiles/common/modules/custom/nexteuropa_lastupdate/nexteuropa_lastupdate.module [hunk 1]
```diff
@@ -23,25 +23,31 @@ function nexteuropa_lastupdate_block_view($delta = '') {
...
 }
 
 /**
+ * Implements hook_preprocess_hook().
+ */
+function nexteuropa_lastupdate_preprocess_nexteuropa_last_update_block(&$variables, $hook) {
+  if ($entity = _nexteuropa_lastupdate_menu_get_any_entity()) {
+    $info = entity_get_info($entity->entity_type);
+    if (isset($info['last update callback']) && function_exists($info['last update callback'])) {
+      $last_update_variables = $info['last update callback']($entity);
```
Comment (drupol) [inline profiles/common/modules/custom/nexteuropa_lastupdate/nexteuropa_lastupdate.module]: I'd use `call_user_func()`.

File: profiles/common/modules/custom/nexteuropa_lastupdate/nexteuropa_lastupdate.module [hunk 2]
```diff
@@ -23,25 +23,31 @@ function nexteuropa_lastupdate_block_view($delta = '') {
...
-            $block_content = theme('nexteuropa_last_update_block', $last_update_variables);
-          }
-        }
-      }
-      if (!isset($block_content)) {
-        $block_content = '';
-      }
-      $block['content'] = $block_content;
+      $block['content'] = array(
+        '#theme' => 'nexteuropa_last_update_block',
```
Comment (drupol) [inline profiles/common/modules/custom/nexteuropa_lastupdate/nexteuropa_lastupdate.module]: 👍 😸

File: profiles/common/modules/custom/nexteuropa_lastupdate/nexteuropa_lastupdate.module [hunk 3]
```diff
@@ -58,6 +64,10 @@ function nexteuropa_lastupdate_theme($existing, $type, $theme, $path) {
     'nexteuropa_last_update_block' => array(
       'template' => 'nexteuropa-last-update-block',
       'path' => $path . '/theme',
+      'variables' => array(
+        'label' => NULL,
+        'date' => NULL,
```
Comment (drupol) [inline profiles/common/modules/custom/nexteuropa_lastupdate/nexteuropa_lastupdate.module]: 👍

File: profiles/common/modules/features/ec_embedded_video/ec_embedded_video.install.inc [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+ * Code for the ec_embedded_video module.
+ */
+
+/**
+ * Soft configure media assets video display for the WYSIWYG view mode.
+ *
+ * Implied media assets are YouTube, Vimeo, Dailymotion and AV portal.
+ */
+function _ec_embedded_video_install_soft_configured_file_display() {
+
```
Comment (drupol) [inline profiles/common/modules/features/ec_embedded_video/ec_embedded_video.install.inc]: Extraneous empty line.

Comment (voidtek) [inline profiles/common/modules/features/ec_embedded_video/ec_embedded_video.install.inc]: Removed.

File: profiles/common/modules/custom/nexteuropa_lastupdate/nexteuropa_lastupdate.module (hunk 1 above)
Comment (voidtek) [inline profiles/common/modules/custom/nexteuropa_lastupdate/nexteuropa_lastupdate.module]: Nice catch, it's changed.

**Comment by drupol** [comment]
👍

File: profiles/common/modules/custom/nexteuropa_lastupdate/theme/nexteuropa-last-update-block.tpl.php [hunk 5]
```diff
@@ -7,14 +7,14 @@
  * Available variables:
  * - $label: the language list.
```
Comment (verbruggenalex) [inline profiles/common/modules/custom/nexteuropa_lastupdate/theme/nexteuropa-last-update-block.tpl.php]: Please update the variable description.

File: profiles/common/modules/custom/nexteuropa_lastupdate/theme/nexteuropa-last-update-block.tpl.php [hunk 6]
```diff
@@ -7,14 +7,14 @@
...
  * - $date: optional language icon.
```
Comment (verbruggenalex) [inline profiles/common/modules/custom/nexteuropa_lastupdate/theme/nexteuropa-last-update-block.tpl.php]: Please update the variable description.

File: profiles/common/modules/custom/nexteuropa_lastupdate/theme/nexteuropa-last-update-block.tpl.php (hunk 5 above)
Comment (verbruggenalex) [inline profiles/common/modules/custom/nexteuropa_lastupdate/theme/nexteuropa-last-update-block.tpl.php]: Please update the variable description.

File: profiles/common/modules/custom/nexteuropa_lastupdate/theme/nexteuropa-last-update-block.tpl.php (hunk 6 above)
Comment (verbruggenalex) [inline profiles/common/modules/custom/nexteuropa_lastupdate/theme/nexteuropa-last-update-block.tpl.php]: Please update the variable description.

File: profiles/common/modules/custom/nexteuropa_lastupdate/theme/nexteuropa-last-update-block.tpl.php (hunk 5 above)
Comment (voidtek) [inline profiles/common/modules/custom/nexteuropa_lastupdate/theme/nexteuropa-last-update-block.tpl.php]: done.

File: profiles/common/modules/custom/nexteuropa_lastupdate/theme/nexteuropa-last-update-block.tpl.php (hunk 6 above)
Comment (voidtek) [inline profiles/common/modules/custom/nexteuropa_lastupdate/theme/nexteuropa-last-update-block.tpl.php]: done.

File: profiles/common/modules/custom/nexteuropa_lastupdate/theme/nexteuropa-last-update-block.tpl.php [hunk 7]
```diff
@@ -5,8 +5,8 @@
  * Default implementation of the last update block.
  *
  * Available variables:
- * - $label: the language list.
- * - $date: optional language icon.
+ * - $label: the language label.
```
Comment (verbruggenalex) [inline profiles/common/modules/custom/nexteuropa_lastupdate/theme/nexteuropa-last-update-block.tpl.php]: :-1: It's not a language label. Last update label has nothing to do with language.
A good description of this variable may be "last updated label"

**Comment by verbruggenalex** [comment]
:+1:

**Comment by voidtek** [comment]
:+1:

**Pull request merged**